login.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Teh Login Page</title>
+
+        <link rel="stylesheet" type="text/css" href="css/login.css">
+
+    </head>
+    <body>
+
+        <div id="external">
+            <div id="login">
+<!--                @Alex I'm not sure, but normally it should have an "action=login.php" or something similar attached-->
```

Comment:
Exactly! Just as long as you're aware of that, our job here is done! For now... :P